UnityProject/Assets/Scripts/Objects/Research/Artifacts/Artifacts.asmdef

```diff
@@ -1,20 +0,0 @@
-{
```

Comment:
Why did you delete the asmdef?

Reply:
Was breaking alot of references for some reason. Maybe it was my IDE? but im using visual studio and unity was complaining too. I don't see why artifacts require their own assembly

Reply:
That's probably because you were trying to use code from outside the boundaries of this assembly definition. Instead of deleting the asmdef I would try finding the way to keep most of this code isolated and maybe share some references. The reason our project takes so much time to compile is because most of the code inst isolated into their own asmdef, so any change no matter how small makes unity compile almost everything. For that reason, we need to make our best to isolate more code instead of moving it into the Assets definition.

Reply:
Having some problems with the asmdef rn, keeps reverting my changes, imma get this pr out of the draft state and then come back to this

Reply:
Just this and the sprite and then this will hopefully be ready to merge.

Reply:
Any update on this, @AtnerNT ?